src/cast_core/src/CrudActor.cpp

```diff
@@ -637,6 +637,9 @@ struct FindOperation : public BaseOperation {
         if (opNode["Options"]) {
             _options = opNode["Options"].to<mongocxx::options::find>();
         }
+        if(opNode["Projection"]){
```

Comment:
One could argue that given the structure of the underlying C++ driver API, the `Projection` should be nested inside `Options` rather than top-level. I suppose that having `Projection` as top-level more closely resembles the actual find command though, so I'm not totally sure what's best. I'd be curious to hear if @rtimmons has a strong opinion here.

Reply:
I have little opinion here. In terms of SEO, find with distinct options and projection is more discoverable and consistent with the shell. This is compelling enough for me. Perhaps we could detect `Projection` inside of `Options` and barf to be helpful?

Reply:
I would prefer to leave this as is if that's alright, as I don't want to keep going back and forth between the two. This matches the CXX driver spec, so unless anyone feels strongly I'm inclined to leave it as is.

Reply:
Sounds like we're all fine with it as currently implemented. Resolving the thread.